package.json

```diff
@@ -62,7 +69,7 @@
     "ts-node": "8.10.2",
     "tslib": "2.2.0",
     "typescript": "3.8.3",
-    "uniforms": "3.5.1",
+    "uniforms": "3.5.2",
     "uniforms-bridge-simple-schema-2": "3.5.1"
```

Comment:
Could you update this dependency to 3.5.2 as well?

Reply:
Yes